src/shared/stores/__tests__/lang.tests.js

```diff
@@ -0,0 +1,118 @@
+import lang from '../lang';
+import * as languages from '../languages';
+
+describe('Theme › Shared › Stores › Lang', () => {
+  test('store has `default`, `current` and `code` props', () => {
+    const self = lang.create();
+
+    expect(self.default).not.toBe(undefined);
+    expect(self.current).not.toBe(undefined);
+    expect(self.code).not.toBe(undefined);
+    expect(self).toMatchSnapshot();
+  });
+
+  test('`default` language should be English', () => {
+    const self = lang.create();
+
+    expect(self.default).toEqual(languages.en);
+  });
+
+  test('default language `code` should be English', () => {
+    const self = lang.create();
+
+    expect(self.code).toBe('en');
+  });
+
+  test('`setLang` should change the value of `current` and `code` to Spanish', () => {
+    const self = lang.create();
+
+    expect(self.current).toEqual({});
+    expect(self.code).toBe('en');
+
+    self.setLang('es');
+
+    expect(self.current).toEqual(languages.es);
+    expect(self.code).toBe('es');
+    expect(self).toMatchSnapshot();
+  });
+
+  test('`setLang` should change the value of `current` to an empty object if the code provided is not available', () => {
+    const self = lang.create({
+      current: languages.es,
+      code: 'es',
+    });
+
+    expect(self.current).toEqual(languages.es);
+    expect(self.code).toBe('es');
+
+    self.setLang('ar');
+
+    expect(self.current).toEqual({});
+    expect(self.code).toBe('ar');
+    expect(self).toMatchSnapshot();
+  });
+
+  test('`get` retrieves the string from the `default` language when that string is not available in `current`', () => {
```

Comment:
Shouldn't you change to a different language than english before retrieving `loadMore`? Maybe to a non-existing language, like `self.setLang('ar');`?

Reply:
I don't think so because there is an empty object in `current`, the same that if I changed to another inexistent language before.

Reply:
Oh, so the `default` logic is having an empty `current` that fallbacks to english. Perfect then 👍